docs/src/reference/gremlin-variants.asciidoc

```diff
@@ -477,24 +485,23 @@ in Gremlin-Java. The `GraphTraversalSource` requires a RemoteConnection implemen
 [source,javascript]
 ----
 const gremlin = require('gremlin');
-const Graph = gremlin.structure.Graph;
+const traversal = gremlin.structure.Graph.traversal;
```

Comment:
This should be exposed at:`gremlin.process.traversal`, to match java's `org.apache.tinkerpop.gremlin.process.traversal`

```suggestion
const traversal = gremlin.process.traversal;
```